docs/backends/clickhouse.qmd

```diff
@@ -77,17 +77,20 @@ con = ibis.clickhouse.connect(
 ```
 
 ::: {.callout-note}
-ibis.clickhouse.connect` is a thin wrapper around [`ibis.backends.clickhouse.Backend.do_connect`][ibis.backends.clickhouse.Backend.do_connect].
+`ibis.clickhouse.connect` is a thin wrapper around [`ibis.backends.clickhouse.Backend.do_connect`](ibis.backends.clickhouse.Backend.do_connect).
```

Comment:
Still need to figure out how to do the interlinks. Looks like a new version of quartodoc (0.6.0) was released that might better support this.

Reply:
Figured this out and it didn't require a version bump, but #7013 will be bumping for other reasons.